easybuild/easyblocks/e/elpa.py

```diff
@@ -176,6 +180,51 @@ def run_all_steps(self, *args, **kwargs):
 
         return super(EB_ELPA, self).run_all_steps(*args, **kwargs)
 
+    def configure_step(self):
+        """Configure step for ELPA"""
+
+        # Add nvidia GPU support if requested
+        cuda_root = get_software_root('CUDA')
+        self.log.info("Got cuda root: %s", cuda_root)
+        if cuda_root:
+            self.cfg.update('configopts', '--enable-nvidia-gpu')
+            self.cfg.update('configopts', '--with-cuda-path=%s' % cuda_root)
+            self.cfg.update('configopts', '--with-cuda-sdk-path=%s' % cuda_root)
+
+            cuda_cc = build_option('cuda_compute_capabilities') or self.cfg['cuda_compute_capabilities']
+            if not cuda_cc:
+                raise EasyBuildError('List of CUDA compute capabilities must be specified, either via '
+                                     'cuda_compute_capabilities easyconfig parameter or via '
+                                     '--cuda-compute-capabilities')
+
+            # ELPA's --with-NVIDIA-GPU-compute-capability only accepts a single architecture
+            if len(cuda_cc) != 1:
+                raise EasyBuildError('ELPA currently only supports specifying one CUDA architecture when '
+                                     'building. You specified cuda-compute-capabilities: %s', cuda_cc)
```

Comment:
should this be a hard error, or should we pick one (e.g. lowest) and throw a warning? I think that's how this is handled in a few other easyblocks

Reply:
I don't think we should assume we know what is best for the user here, so I'd personally prefer an error. That allows the user to pick by simply passing a single compute capability on the command line. If I _know_ that only a single one is supported, I could decide to pick the highest and only run on nodes that support that architecture.

Reply:
(or, obviously, I could decide to pick the lowest and be able to run on any GPU node in that system)

Reply:
there's something to say about both approaches,
but imho that discussion should not block this PR.